DrawerController/DrawerController.swift

```diff
@@ -1511,42 +1511,37 @@ public class DrawerController: UIViewController, UIGestureRecognizerDelegate {
+        var navigationBarRect = CGRect.null
         
         if let centerViewController = self.centerViewController {
-            if centerViewController.isKindOfClass(UINavigationController) {
+            if centerViewController.isKind(of: UINavigationController.self) {
                 let navBar = (self.centerViewController as! UINavigationController).navigationBar
-                navigationBarRect = navBar.convertRect(navBar.bounds, toView: self.childControllerContainerView)
-                navigationBarRect = CGRectIntersection(navigationBarRect, self.childControllerContainerView.bounds)
+                navigationBarRect = navBar.convert(navBar.bounds, to: self.childControllerContainerView)
+                navigationBarRect = navigationBarRect.intersection(self.childControllerContainerView.bounds)
             }
         }
         
-        return CGRectContainsPoint(navigationBarRect, point)
+        return navigationBarRect.contains(point)
     }
     
-    func isPointContainedWithinCenterViewContentRect(point: CGPoint) -> Bool {
+    func isPointContainedWithinCenterViewContentRect(_ point: CGPoint) -> Bool {
         var centerViewContentRect = self.centerContainerView.frame
-        centerViewContentRect = CGRectIntersection(centerViewContentRect, self.childControllerContainerView.bounds)
+        centerViewContentRect = centerViewContentRect.intersection(self.childControllerContainerView.bounds)
         
-        return CGRectContainsPoint(centerViewContentRect, point) && self.isPointContainedWithinNavigationRect(point) == false
+        return centerViewContentRect.contains(point) && self.isPointContainedWithinNavigationRect(point) == false
     }
     
-    func isPointContainedWithinLeftBezelRect(point: CGPoint) -> Bool {
-        var leftBezelRect = CGRectNull
-        var tempRect = CGRectNull
+    func isPointContainedWithinLeftBezelRect(_ point: CGPoint) -> Bool {
+        let leftBezelRect = CGRect.null
+        _ = childControllerContainerView.bounds.divided(atDistance: bezelRange, from: .minXEdge)
         
-        CGRectDivide(self.childControllerContainerView.bounds, &leftBezelRect, &tempRect, bezelRange, .MinXEdge)
-        
-        return CGRectContainsPoint(leftBezelRect, point) && self.isPointContainedWithinCenterViewContentRect(point)
+        return leftBezelRect.contains(point) && self.isPointContainedWithinCenterViewContentRect(point)
     }
     
-    func isPointContainedWithinRightBezelRect(point: CGPoint) -> Bool {
-        var rightBezelRect = CGRectNull
-        var tempRect = CGRectNull
-        
-        CGRectDivide(self.childControllerContainerView.bounds, &rightBezelRect, &tempRect, bezelRange, .MaxXEdge)
-        
-        return CGRectContainsPoint(rightBezelRect, point) && self.isPointContainedWithinCenterViewContentRect(point)
+    func isPointContainedWithinRightBezelRect(_ point: CGPoint) -> Bool {
+        let rightBezelRect = CGRect.null
```

Comment:
shouldn't it rather be:
`var rightBezelRect = CGRect.null`
`(rightBezelRect, _) = self.childControllerContainerView.bounds.divided(atDistance: bezelRange, from: .maxXEdge)`